src/reference/asciidoc/jdbc.adoc

```diff
@@ -575,6 +578,29 @@ The message data for a persistent channel is keyed in the store on the channel n
 Consequently, if the channel names are not globally unique, the channels can pick up data that is not intended for them.
 To avoid this danger, you can use the message store `region` to keep data separate for different physical channels that have the same logical name.
 
+==== PostgreSQL: receiving push notifications
+
+PostgreSQL offers a listen and notification framework for receiving push notifications upon database table manipulations.
+Spring Integration leverages this mechanism to allow for receiving push notifications when new messages are added to a
+`JdbcChannelMessageStore`. When using this feature, a database trigger must be defined, which can be found as part of the
+comments of the `schema-postgresql.sql` file which is included in the JDBC module of Spring Integration.
+
+Push notifications are received via the `PostgresChannelMessageTableSubscriber` class which allows its subscribers to
+receive a callback upon the arrival of new messages for any given `region` and `groupId`. These notifications are received
+even if a message was appended on a different JVM, but to the same database. To read messages when they arrive, a
+`PostgresSubscribableChannel` can delegate new messages to a `MessageHandler` what allows to implement an effective
+pub-sub mechanism based on JDBC.
```

Comment:
I guess would be great to add a simple sample of the possible configuration of the channel and that `PostgresChannelMessageTableSubscriber` .
As `@Bean`s, of course! 😄